cookbooks/fb_stunnel/README.md

```diff
@@ -0,0 +1,62 @@
+fb_stunnel Cookbook
+===================
+Manage stunnel service and configuration
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_stunnel']['enable']
+* node['fb_stunnel']['config']
+* node['fb_stunnel']['config'][$CONFIG]
+* node['fb_stunnel']['sysconfig'][$CONFIG]
+
+Usage
+-----
+### Config
+The config is mapped 1:1 to an INI file with the exception of the `_create_self_signed_cert` key described below. The top-level key is the INI section, and below that are expected to be simple key-value pairs. For example:
```

Comment:
can you pass this through markdownlint? I'm getting a bunch of errors when importing it